n_grasps: 20
n_viewpoints: 10
vol_res: 80
